app/controllers/application_controller.rb

```diff
@@ -3,6 +3,8 @@ class ApplicationController < ActionController::Base
   # For APIs, you may want to use :null_session instead.
   protect_from_forgery with: :exception
 
+  before_action :do_not_cache
```

Comment:
Do we actually want to apply this application-wide? We probably do as it stands, but if we have to host the staff info pages, we should probably remember to remove this.

Also, does this apply to assets? If not, then we're fine, but we should probably allow those to be cached as heavily as possible…